src/App.tsx

```diff
@@ -58,6 +60,11 @@ export default class App extends React.Component<
     };
 
     updateTheme(defaultTheme);
+    this.onFileNameChangeHandler = this.onFileNameChangeHandler.bind(this);
```

Comment:
Do you mind updating this to use class field syntax and making it private?

Reply:
I used an arrow function here, hope this works as well for you